web/web.go

```diff
@@ -67,6 +68,29 @@ import (
 
 var localhostRepresentations = []string{"127.0.0.1", "localhost"}
 
+var (
+	requestDuration = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "http_request_duration_seconds",
```

Comment:
prometheus_http_request_duration_seconds

Reply:
@brian-brazil What's the reason for prefixing `http_request_duration` with  `prometheus` now? I consider it as a non-Prometheus specific metric and it is a [standardized best practice](https://github.com/prometheus/docs/blob/master/content/docs/practices/naming.md#metric-names).

Reply:
These metrics are coming from the "prometheus library" and thus should be prefixed accordingly, so they don't clash with http metrics from other libraries.

Reply:
The meaning of a `http_request_duration` shouldn't differ no matter which library creates it, shall it? I think a label like `metric_producer="client_golang"` is sufficient to bug-hunt in case a library offers values with different meaning. That way, looking at Prometheus metrics could still serve as best practice 👍
Prometheus (normally) adds a `job="prometheus"` label to the metric anyway.

Reply:
The meaning of a `http_request_duration` shouldn't differ no matter which library creates it, shall it? I think a label like `metric_producer="client_golang"` is sufficient to bug-hunt in case a library offers values with different meaning. That way, looking at Prometheus metrics could still serve as best practice 👍
Prometheus (normally) adds a `job="prometheus"` label to the metric anyway.

Reply:
I would be extremely surprised if it didn't differ. Each library has different semantics, different labels, and different middleware. The same request might hit multiple such libraries. Accordingly this does not make sense as one metric separated by labels.
In addition a metric should almost always live inside one file, largely for the above reasons.